src/new_to_cabal/01_installing_cabal_and_ghc.md

```diff
@@ -1 +1,73 @@
 # Installing cabal + GHC
+
+## ghcup
+
+While most package managers have some support for installing at least ghc (and
```

Comment:
Perhpas "managers (notably cholocatey[link[ for Windows)" until ghcup support for Windows reaches the level chocolatey provides?